code/modules/clothing/glasses/glasses.dm

```diff
@@ -136,6 +136,13 @@
 	item_state = "glasses"
 	vision_correction = 1 //corrects nearsightedness
 
+/obj/item/clothing/glasses/regular/jamjar
+	name = "Jamjar Glasses"
+	desc = "Also known as Virginity Protectors."
+	icon_state = "jamjar_glasses"
+	item_state = "jamjar_glasses"
+	vision_correction = 1 //corrects nearsightedness

```

Comment:
How does that work? Its position in the file after the original prescription glasses, or something I'm missing? Everything I've done so far is based off existing code, so I'm a little afraid to deviate.